hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java

```diff
@@ -84,6 +86,13 @@ public HollowClientUpdater(HollowConsumer.BlobRetriever transitionCreator,
         this.initialLoad = new CompletableFuture<>();
     }
 
+    public void setSkipShardUpdateWithNoAdditions(boolean skipTypeShardUpdateWithNoAdditions) {
```

Comment:
Do you think it would be good to have a similar method on the `HollowConsumer` as well so a consumer can toggle this behavior on the fly?

Reply:
Hmm, yes maybe.  That way we could potentially flip back with FP if we catch something in the integrity check.